### PR TITLE
[15.0][MIG] web_ir_actions_act_view_reload: Migration to 15.0

### PR DESCRIPTION
**Pull request by manuelcalerosolis**
This is the migration of ir_actions_act_view_reload to 15.0

### CODE REVIEW

File: web_ir_actions_act_view_reload/static/src/js/web_ir_actions_act_view_reload.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+                return this._super.apply(this, arguments);
+            },
+
+            /**
+             * Handle 'ir.actions.act_view_reload' action
+             * @returns {Promise} Resolved promise
+             */
+            _executeReloadAction: function () {
+                var controller = this.getCurrentController();
```
Comment (StefanRijnhart) [inline web_ir_actions_act_view_reload/static/src/js/web_ir_actions_act_view_reload.js:29]: Does https://github.com/OCA/web/pull/2241 apply here as well?

**Comment by StefanRijnhart** [comment]
This PR https://github.com/OCA/web/pull/2192 implies that the module breaks in some cases if the action is not defined as a model. Do you want to review it, and check if this is still true in Odoo 15?

**Comment by janikvonrotz** [comment]
@StefanRijnhart I was trying to setup `project_timesheet_time_control` using this and other PRs:

```bash
# web_ir_actions_act_view_reload
cd oca/web
gh pr checkout 2240
../..
# hr_timesheet_task_stage
cd oca/timesheet
gh pr checkout 500
../..
# project_timesheet_time_control
cd oca/project
gh pr checkout 939
../..
```
For some reason the action `ir_actions_act_view_reload` cannot be loaded.

grafik

**Comment by StefanRijnhart** [comment]
@janikvonrotz That would be useful feedback to the author of this PR, @manuelcalerosolis

**Comment by janikvonrotz** [comment]
@manuelcalerosolis The client throws this error:

```
UncaughtPromiseError
Uncaught Promise > The ActionManager service can't handle actions of type ir.actions.act_view_reload
Error: The ActionManager service can't handle actions of type ir.actions.act_view_reload
    doAction@http://localhost:8069/web/assets/debug/web.assets_backend.js:24916:23 (/web/static/src/webclient/actions/action_service.js:1132)
```

**Comment by StefanRijnhart** [comment]
Wow, it looks like the entry point `_handleAction` does not exist in Odoo 15 anymore (https://github.com/odoo/odoo/commit/0573acae2306bf5da2005852da9323ddc59e5431#diff-b4ba6932eee92edebdb85d63cbd875c0af5bce6532286faa30de6586f72c1cc5), and was replaced by https://github.com/odoo/odoo/blob/15.0/addons/web/static/src/webclient/actions/action_service.js#L1106-L1137.

@manuelcalerosolis did this code work for you locally on Odoo 15?

**Comment by manuelcalerosolis** [comment]
> Wow, it looks like the entry point `_handleAction` does not exist in Odoo 15 anymore ([odoo/odoo@0573aca#diff-b4ba6932eee92edebdb85d63cbd875c0af5bce6532286faa30de6586f72c1cc5](https://github.com/odoo/odoo/commit/0573acae2306bf5da2005852da9323ddc59e5431#diff-b4ba6932eee92edebdb85d63cbd875c0af5bce6532286faa30de6586f72c1cc5)), and was replaced by https://github.com/odoo/odoo/blob/15.0/addons/web/static/src/webclient/actions/action_service.js#L1106-L1137.
> 
> @manuelcalerosolis did this code work for you locally on Odoo 15?

@StefanRijnhart I also get the error when starting a timesheet within HelpDesk with the "Start Work" button.

I really don't know what change I have to incorporate in the javascript module so that it works correctly in 15.0

**Comment by StefanRijnhart** [comment]
Then maybe you should leave the migration of this module to someone else.

**Comment by manuelcalerosolis** [comment]
> Then maybe you should leave the migration of this module to someone else.

Sure. 

Maybe that person is you.

**Comment by janikvonrotz** [comment]
@StefanRijnhart Is there really no alternative in v15?

Something like: `{"type": "ir.actions.client", "tag": "reload"}`

**Comment by pedrobaeza** [comment]
Superseded by #2272

**Pull request closed without merging**